Client/Assets/SearchPlugins/yahoo-br.xml

```diff
@@ -1,22 +0,0 @@
-<!-- This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
these regional files are not needed anymore, I manually add regional subdomain to the query in `OpenSearch.swift`